website/blog/2023-06-11-reborn.md

```diff
@@ -0,0 +1,56 @@
+---
+title: ReasonReact is back (now with Melange)
+---
+
+Since the ReScript project diverged from ReasonML and moved ReasonReact into `@rescript/react`, ReasonReact has been in limbo. While this happened, the Melange team released the first major release of [Melange](https://buttondown.email/anmonteiro/archive/melange-hits-v10/) and made the first steps towards a deeper integration with the OCaml ecosystem.
```

Comment:
```suggestion
ReasonReact moved to `@rescript/react` after the ReScript announcement. The future of ReasonReact has been uncertain ever since. In the meantime, the Melange team announced the first major release of [Melange](https://anmonteiro.substack.com/p/melange-10-is-here) and took the first steps towards deeper integration with the OCaml ecosystem.
```